### ISSUE TITLE
Is this project dead

### ISSUE THREAD

**Issue by PieterEbit** [greenfieldtech-nirs/phpari#43]
I'm asking as php-react has made some leaps, and there are a few small things I want to update and fix.

**Comment by greenfieldtech-nirs** [comment]
It's not dead, you are welcome to offer a pull request. Simply real life
work take precedent.

**Comment by PieterEbit** [comment]
Cool will do, just wanted to make sure. 

The one thing I want to touch and this might be a breaking change, at the moment you are using [pest](https://github.com/educoder/pest) now the issue is it uses curl and does block io something we don not want to happen, so i'll change that to use react-guzzle and then things like the "channel id" that gets returned by the originate will be a promise. do I branch from master or your develop. I see you've started with adding asterisk 14 things to it.

**Comment by greenfieldtech-nirs** [comment]
Branch from develop

**Comment by greenfieldtech-nirs** [comment]
Hi There,

  I've created a new branch to experiment with Guzzle HTTP. At face value, PEST seems much simpler, and for the general usage - it seems like a proper solution. However, I'm open to experimenting with Guzzle. 
  
  I've modified the "application" GET method to use guzzle for now, simply as a basic test. If you have a recommendation as to how to utilize it better - I'm open.

Nir
